app/controllers/users_controller.rb

```diff
@@ -1,4 +1,13 @@
 class UsersController < ApplicationController
+  before_action :find_user, except: %i(index new create)
+  before_action :logged_in_user, only: %i(index edit update destroy)
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i(index edit update destroy)
  before_action :find_user, except: %i(index new create)
```
ri hợp lý hơn, có quyền đó mới cần phải load data, ko có quyền load làm j